lib/fluent/plugin/filter_throttle.rb

```diff
@@ -0,0 +1,137 @@
+require 'fluent/filter'
+
+module Fluent
+  class ThrottleFilter < Filter
+    Fluent::Plugin.register_filter('throttle', self)
+
+    config_param :group_key, :string, :default => 'kubernetes.container_name'
```

Comment:
Should it be container_name or pod_name? Jobs may have multiple containers.

Reply:
My assumption was that we most likely want to throttle for all replicas the same way.

Note that this is only the default and it can be changed via configuration.

But we can set pod_name here if you want.